docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3'
+services:
+  app:
+    image: node:8.11
```

Comment:
I think we should move up to Node 10 which is the current LTS. Consider also not having the pin to the minor version here, just `node:10`. We could also probably use alpine or slim here.

Reply:
Makes sense. I'll switch to `node:10`

Reply:
Updated.